docs/config/volume-support.md

```diff
@@ -0,0 +1,206 @@
+# volume-support
+
+> Beta
+>
+> This kubeflow pipeline volume support has **beta** status 
```

Comment:
Hi @haibingzhao, sorry every feature must start from alpha.
After letting some early adopters use it successfully for a while, we can push it to Beta.